src/frontend/src/features/layouts/components/mailbox-settings/modal-mailbox-settings/_index.scss

```
@@ -0,0 +1,156 @@
+    // instead of overflowing the modal.
+    overflow-y: auto;
+}
+
+.mailbox-settings {
+  // macOS-style account card sitting at the very top of the settings sidebar:
+  // mailbox avatar on the left, name in bold and (when set) the address in a
+  // muted line underneath.
+  &__identity {
+    /* gap: var(--c--globals--spacings--xs); */
+    width: 100%;
+    // Min-width:0 lets the flexible text column actually shrink so the long
```

Stylelint (17.12.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)